pkg/ssh/ssh.go

```diff
@@ -82,6 +120,50 @@ func execSSHCommand(provider *SSHProvider, command string, output io.Writer) err
 	return cmd.Run()
 }
 
+func copyCommandToRemote(provider *SSHProvider, command string) (string, error) {
+	script, err := os.CreateTemp("/tmp/", "devpod-command-*")
```

Comment:
why specify `/tmp/` and not let the os figure out the preferred temporary directory?

Reply:
righty, done :+1: 

it will use OS's native one on the host side, but still forse /tmp for the destination in container tho